<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressionGroupsDialog</class>
 <widget class="QDialog" name="ProgressionGroupsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Progression Locations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="vlay_progression_locations">
     <item>
      <layout class="QHBoxLayout" name="hlay_progression_locations_body">
       <item>
        <layout class="QVBoxLayout" name="vlay_disabled_locations">
         <item>
          <widget class="QLabel" name="label_disabled_locations">
           <property name="text">
            <string>Non-Progression Locations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="disabled_locations_free_search">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="disabled_locations">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionRectVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlay_locations_controls_disable">
         <item>
          <spacer name="vspace_tricks_controls_upper1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="disable_location">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Disable
&lt;--</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="vspace_tricks_controls_middle1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="randomize_location1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Randomize
--&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="vspace_tricks_controls_lower1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlay_randomized_locations">
         <item>
          <widget class="QLabel" name="label_randomized_locations">
           <property name="text">
            <string>Randomized Locations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="randomized_locations_free_search">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="randomized_locations">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlay_locations_controls_enable">
         <item>
          <spacer name="vspace_tricks_controls_upper2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="randomize_location2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Randomize
&lt;--</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="vspace_tricks_controls_middle2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="enable_location">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable
--&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="vspace_tricks_controls_lower2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlay_enabled_locations">
         <item>
          <widget class="QLabel" name="label_enabled_locations">
           <property name="text">
            <string>Enabled Progression Locations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enabled_locations_free_search">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="enabled_locations">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionRectVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="dialog_description">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: grey;</string>
       </property>
       <property name="text">
        <string>Enabled progression locations can have progress items. Randomized locations can be progression or non-progression, which will be randomly selected when
the seed is generated. Individual locations that are &quot;excluded locations&quot; on the main page will remain non-progression locations.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bbox_progression_locations">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbox_progression_locations</sender>
   <signal>accepted()</signal>
   <receiver>ProgressionGroupsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbox_progression_locations</sender>
   <signal>rejected()</signal>
   <receiver>ProgressionGroupsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
